spec/dangerfile_spec.rb

```diff
@@ -33,6 +33,31 @@ def make_temp_file(contents)
     end.to raise_error(Danger::DSLError)
   end
 
+  it 'respects ignored violations' do
+    code = "message 'A message'\n" +
+           "warn 'An ignored warning'\n" +
+           "warn 'A warning'\n" +
+           "fail 'An ignored error'\n" +
+           "fail 'An error'\n"
```

Comment:
I believe this is the only code style thing here, and it's also the only think keeping the red lights on, apart from this, should be good, right?